projects/igniteui-angular/src/lib/carousel/carousel.component.ts

```diff
@@ -106,6 +106,14 @@ export class IgxCarouselComponent implements OnDestroy, AfterContentInit {
      */
     @HostBinding('attr.role') public role = 'region';
 
+    /** @hidden */
+    @HostBinding('attr.aria-roledescription')
+    public roleDescription = 'carousel';
+
+    /** @hidden */
+    @HostBinding('attr.aria-labelledby')
+    public labelId = `igxCarouselLbl`; // tab-${this.current + 1}-3
```

Comment:
Should this be static?

Reply:
yep, this is something that we discussed while she was addressing some in-progress stuff, this will become:

```
public labelId = `${this.id}-label`;
```

the template will be updated accordingly